### PR TITLE
fix: update circular progress percentage color

### LINKED ISSUE #2178: ProgressBar component color issues
Please check below issues in ProgressBar component:

1. Percentage text has black color. expected: gray.normal in dark mode.
2. Even though isIndeterminate is false, progress bar color is blinking.

code : 
```
<ProgressBar
  showPercentage
  isIndeterminate={false}
  size="large"
  variant="circular"
  value={(budget.settled_amount / budget.amount) * 100}
  color="positive"
/>
```

image

Design link - https://www.figma.com/design/uLiXKA43fEOUwU7Fw2eodL/Budgets?node-id=345-172373&m=dev

### PR DESCRIPTION
**Pull request by snitin315**
Fix #2178

### CODE REVIEW

File: packages/blade/src/components/ProgressBar/CircularProgressBar.web.tsx [hunk 1]
```diff
@@ -60,10 +60,13 @@ const getPulseAnimationStyles = ({
...
-    return getBaseTextStyles({
-      theme,
-      ...getTextProps({ variant: 'body', size, weight }),
-    });
+    return {
+      ...getBaseTextStyles({
+        theme,
+        ...getTextProps({ variant: 'body', size, weight }),
+      }),
+      fill: getIn(theme.colors, 'surface.text.gray.normal'),
```
Comment (snitin315) [inline packages/blade/src/components/ProgressBar/CircularProgressBar.web.tsx:68]:
```suggestion
      fill: theme.colors.surface.text.gray.normal
```

File: packages/blade/src/components/ProgressBar/CircularProgressBar.web.tsx [hunk 2]
```diff
@@ -60,10 +60,13 @@ const getPulseAnimationStyles = ({
...
-    return getBaseTextStyles({
-      theme,
-      ...getTextProps({ variant: 'body', size, weight }),
-    });
+    return {
+      ...getBaseTextStyles({
+        theme,
+        ...getTextProps({ variant: 'body', size, weight }),
+      }),
+      fill: theme.colors.surface.text.gray.normal
```
Comment (snitin315) [inline packages/blade/src/components/ProgressBar/CircularProgressBar.web.tsx:68]:
```suggestion
      fill: theme.colors.surface.text.gray.normal,
```

File: packages/blade/src/components/ProgressBar/CircularProgressBar.web.tsx [hunk 3]
```diff
@@ -19,13 +19,13 @@ import getBaseTextStyles from '~components/Typography/BaseText/getBaseTextStyles
 
 const pulseKeyframes = keyframes`
   0% {
-    opacity: 100;
```
Comment (anuraghazra) [inline packages/blade/src/components/ProgressBar/CircularProgressBar.web.tsx:22]: wow how did this even happen?

Comment (snitin315) [reply packages/blade/src/components/ProgressBar/CircularProgressBar.web.tsx:22]: I just saw this & fixed it. This wasn't causing any issues though.

**Review by anuraghazra — APPROVED** [review]

**Pull request merged**